Core/Core/Store/Store.swift

```diff
@@ -35,7 +35,10 @@ public class Store<U: UseCase>: NSObject, NSFetchedResultsControllerDelegate {
     public let eventHandler: EventHandler
 
     public var count: Int {
-        return numberOfObjects(inSection: 0)
+        guard let sections = frc.sections, sections.count > 0 else {
+            return 0
+        }
+        return sections[0].numberOfObjects
```

Comment:
this is to avoid a crash, right? Would `sections.first?.numberOfObjects ?? 0` work?

Reply:
Oh that's a great idea.